### ISSUE TITLE
Unable to pull some extensions.

### ISSUE THREAD

**Issue by c-edw** [c-edw/extensionman#1]
For example, https://chrome.google.com/webstore/detail/https-everywhere/gcbommkclmclpchllfjekcdonpmejbdp/.

**Comment by c-edw** [comment]
Ran Fiddler to see what URL the browser is fetching when installing HTTPS Everywherer from the Chrome Web Store:

```
/service/update2/crx?response=redirect&os=win&arch=x64&os_arch=x86_64&nacl_arch=x86-64&prod=chromiumcrx&prodchannel=&prodversion=71.0.3578.98&lang=en-US&acceptformat=crx2,crx3&x=id%3Dgcbommkclmclpchllfjekcdonpmejbdp%26installsource%3Dondemand%26uc
```

**Comment by c-edw** [comment]
It looks like all that I need add to the fetch URL is `&acceptformat=crx2,crx3`.

**Comment by c-edw** [comment]
Fixed by a65df1e8734fd7ba3034fb91464ff5fc737e4127.
